server/handlers/validation_test.go

```diff
@@ -197,11 +395,53 @@ func TestValidateRootRotation(t *testing.T) {
 
 	updates := []storage.MetaUpdate{root, targets, snapshot, timestamp}
 
-	copyTimestampKey(t, repo, store, "testGUN")
-	_, err = validateUpdate(crypto, "testGUN", updates, store)
+	serverCrypto := copyKeys(t, crypto, data.CanonicalTimestampRole)
+	_, err = validateUpdate(serverCrypto, "testGUN", updates, store)
 	assert.NoError(t, err)
 }
 
+func TestInvalidRootRotation(t *testing.T) {
```

Comment:
Can we add a comment as to in what way the root is invalid? It appears to be because the root is signed with only the new key and not the old key?